include/swift/Reflection/ReflectionContext.h

```diff
@@ -1626,15 +1679,33 @@ class ReflectionContext
       return {std::string("failure reading actor"), {}};
 
     ActorInfo Info{};
-    Info.Flags = ActorObj->Status.Flags[0];
 
-    // Status is the low 3 bits of Flags. Status of 0 is Idle. Don't read
-    // FirstJob when idle.
-    auto Status = Info.Flags & 0x7;
-    if (Status != 0) {
+    enum : uint32_t {
+      ActorStateMask = 0x7,
+
+      Idle = 0x0,
+
+      DistributedRemote = 0x8,
+      IsPriorityEscalated = 0x10,
+
+      PriorityMask = 0xFF00,
+      PriorityShift = 0x8,
+    };
```

Comment:
Could we hoist the enum out of the function scope?  Having the symbolic names for the flags is a general convenience (and may be useful outside of the function scope).

Reply:
This feels like it ought to be shared with `ActiveActorStatus` in some way. Similarly for the flags for the task status.